README.md

```diff
@@ -299,6 +299,38 @@ const { form, status } = useForm({
 </p>
 </details>
 
+<details><summary>Lazy rule validation</summary>
+<p>
+
+You can set `lazy` to `true` to prevent rules from being automatically verified when data changes.
+
+In this case, consider call `verify()` or `status[fieldName].verify()` to manually validate the rule.
```

Comment:
```suggestion
In this case, consider call `verify()` or `status[fieldName].verify()` to manually validate fields.
```

Reply:
Perhaps you mean **validate fields** not **the rule**? Since rules will never require validations.